source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -15,6 +15,16 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 4.3
```

Comment:
This is still beta, so we shouldn't include 4.3 yet. For context, I think we made this agreement not to document beta versions in compatibility tables in a past Drivers Engineering Leads meeting.